src/lang/en-US.ts

```diff
@@ -141,6 +143,7 @@ const CTAs: ResolvedIntlConfig['messages'] = {
     'cta.showAll': `Show All`,
     'cta.reload': `Reload`,
     'cta.evolve': `Apply`,
+    'cta.support': `contact support`,
```

Comment:
This CTA appears in six locations outside of `'error.instructions'`. Were you thinking of replacing them separately?

Reply:
Yeah - gonna switch 'em over later on.